pkg/controller/servicebindingrequest/binding.go

```diff
@@ -220,6 +220,15 @@ func (b *ServiceBinder) onError(
 func (b *ServiceBinder) Bind() (reconcile.Result, error) {
 	sbrStatus := b.SBR.Status.DeepCopy()
 
+	if b.Secret.plan.SBR.Spec.BindingReference == nil {
+		b.Secret.plan.SBR.Spec.BindingReference = &v1alpha1.BindingReference{
+			ObjectType: v1.GroupVersionKind{
+				Kind:    SecretKind,
+				Version: "v1",
+			},
+		}
+	}
+
 	b.Logger.Info("Saving data on intermediary secret...")
 	secretObj, err := b.Secret.Commit(b.Data)
```

Comment:
We can track in a different issue to show the configmap in the status when intermediate secret is not being used. Currently, we have info for binding secret

Reply:
Very good point.

Reply:
Here you go https://github.com/redhat-developer/service-binding-operator/pull/366/commits/1c07fe2bde943393054d1c76c4119a2fc5687b15 . It would be unfair to not include the changes in this PR, I think.